TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,198 @@
+
+The bounding volume for the tile must be a tile bounding region containing the north, south, east, and west bounds of the tile. The positions are represented by u, v, and height values that are quantized and delta encoded.
+
+| Field | Meaning |
+| --- | --- |
+| u | The horizontal coordinate of the vertex in the tile. When the u value is 0, the vertex is on the Western edge of the tile. Then the value is 32767, the vertex is on the Eastern edge of the tile. For other values, the vertex's longitude is a linear interpolation between the longitudes of the Western and Eastern edges of the tile. |
+| v | The vertical coordinate of the vertex in the tile. When the v value is 0, the vertex is on the Southern edge of the tile. When the value is 32767, the vertex is on the Northern edge of the tile. For other values, the vertex's latitude is a linear interpolation between the latitudes of the Southern and Northern edges of the tile. |
+| height | The height of the vertex of the tile. When the height value is 0, the vertex's height is equal to `MINIMUM_HEIGHT` from the feature table. When the value is 32767, the vertex's height is equal to `MAXIMUM_HEIGHT` from the feature table. For other values, the vertex's height is a linear interpolation of the minimum and maximum heights. |
+
+The values are then delta and ZigZag encoded. The delta encoding ensures the values are small integers. The ZigZag encoding ensure the values are positive integers. Example encoding code is listed below:
+```javascript
+function zigZag(value) {
+    return ((value << 1) ^ (value >> 15)) & 0xFFFF;
+}
+
+var lastU = 0;
+var lastV = 0;
+var lastHeight = 0;
+
+for (var i = 0; i < length; ++i) {
+    var u = uBuffer[i];
+    var v = vBuffer[i];
+    var height = heightBuffer[i];
+    
+    uBuffer[i] = zigZag(u - lastU);
+    vBuffer[i] = zigZag(v - lastV);
+    heightBuffer = zigZag(height - lastHeight);
+
+    lastU = u;
+    lastV = v;
+    lastHeight = height;
+}
+```
+
+Example decoding code is listed below:
+```javascript
+function zigZagDecode(value) {
+    return (value >> 1) ^ (-(value & 1));
+}
+
+var u = 0;
+var v = 0;
+var height = 0;
+
+for (var i = 0; i < length; ++i) {
+    u += zigZagDecode(uBuffer[i]);
+    v += zigZagDecode(vBuffer[i]);
+    height += zigZagDecode(heightBuffer[i]);
+    
+    uBuffer[i] = u;
+    vBuffer[i] = v;
+    heightBuffer[i] = height;
+}
+```
+
+#### Polygon positions
+
+The first section of the positions buffer, from offset 0 to `polygonPositionsByteLength`, contains the polygon positions. Polygon positions only have u and v values. The u values are from offset `0` to `polygonPositionsByteLength / 2`. The v values are from offset `polygonPositionsByteLength /2` to `polygonPositionsByteLength`. The number of positions for each polygon is determined by the value of its `POLYGON_COUNT`. 
```

Comment:
For each of these three sections, perhaps replace/augment the first sentence with a small table showing the start byte length and end byte length.